test/Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -199,6 +199,18 @@ spec =
             e_tpe . should_equal_type IOException
             e_tpe . should_not_equal_type JException
 
+        Test.specify "fields of a Type" <|
+            typ = Boolean
+
+            Meta.is_atom typ . should_be_true
+            meta_typ = Meta.meta typ
+            meta_typ . should_be_a Meta.Atom
+            fields = case meta_typ of
+                Meta.Atom.Value _ -> meta_typ.constructor.fields
+                _ -> Test.fail "Should be a Meta.Atom.Value: " + meta_typ.to_text
+
+            fields . should_equal []
```

Comment:
Can we also add a test where `typ` is not a regular type but a module, like `Meta`?

[As mentioned](https://github.com/enso-org/enso/issues/5805#issuecomment-1465822727), this also used to fail on modules and I'd like to ensure it doesn't crash there too.

Looks like the fix should apply there just fine, but good to have a test to avoid regressions.

Reply:
The fallback applies to `Module` as well: https://github.com/enso-org/enso/pull/5869/files#diff-63d4533fccf2dfeb8e87e9ccba516458b86d82dd6f4ee9d9a4b7cc6dec4e667bR35 - too late to do changes to the #5869 PR, but feel free to add a test in its own PR.